setup.cfg

```diff
@@ -7,3 +7,10 @@ pep8ignore = *.py E402 W503 E203 E231 E251 E701
 
 [mypy]
 ignore_missing_imports = True
+
+[pytest]
+markers =
+    skipMlFramework: marks a test to be skipped for specific mlFramework values
+    only_with_platform:  marks a test to be performed only for a specific mlFramework values
```

Comment:
Should we list in the explanation of the markers the possible values for the frameworks?

Reply:
yes that's a good idea will do that :-)